src/main/helm/helm-chart-manager.ts

```diff
@@ -24,7 +24,7 @@ import v8 from "v8";
 import * as yaml from "js-yaml";
 import type { HelmRepo } from "./helm-repo-manager";
 import logger from "../logger";
-import { promiseExecFile } from "../promise-exec";
+import { promiseExecFile } from "../../common/utils/promise-exec";
```

Comment:
nit: `common/utils/index.ts` should have `export * from "./promise-exec";`

Reply:
Let's do this on a separate PR.